modules/default/calendar/calendar.js

```diff
@@ -228,7 +229,7 @@ Module.register("calendar", {
 							// This event falls within the config.urgency period that the user has set
 							timeWrapper.innerHTML = this.capFirst(moment(event.startDate, "x").fromNow());
 						} else {
-							timeWrapper.innerHTML = this.capFirst(moment(event.startDate, "x").format(this.config.dateFormat));
+							timeWrapper.innerHTML = this.capFirst(moment(event.startDate, "x").format(this.config.fullDayEventDateFormat));
```

Comment:
Example of current calendar (left) and after my change (right):

![mirrorcal](https://cloud.githubusercontent.com/assets/25268023/25557779/b617e1a4-2d18-11e7-9750-a33837282705.jpg)